### DISCUSSION TITLE
Dawn/Day buffer

### DISCUSSION THREAD

**Discussion by kme77** [nathanmarlor/foxess_em#125] [Q&A] [answered]
I'm probably just being an idiot but could you expand on how the dawn/Day buffer works. I've been messing around with it but I'm still unsure?

I assumed that it reserved the amount charge you set so it never dropped below that amount, is that correct? If so I think I understand the dawn buffer (i.e. overnight it won't drop below that charge before dawn) but how is the day buffer used and how do the two buffers interact?
Cheers!

**Comment by nathanmarlor** [comment, marked as answer]
@kme77 

Dawn buffer - how much capacity to ensure is left before the solar takes over from the battery (dawn time).
Day buffer - how much capacity to ensure is left at the end of the day.

I implemented the dawn buffer as during the summer months you want to ensure you're charging just enough to get through to reasonable solar output - however, as the forecast can change, condensation on the panels etc. it became clear that adding a small buffer helped avoid completely emptying the battery before the solar takes over.

The day buffer is similar, in that as the forecast, usage etc. changes you want to ensure you're not emptying the battery before you reach the off-peak period.

I have these set to the defaults (dawn - 1, day - 2) in my system and find they work reasonably well.

Hope this helps!

**Reply by kme77** [reply]
Thanks that's cleared it up, cheers! 👍
